src/test/acceptance/features/check-details.feature

```diff
@@ -25,7 +25,7 @@ Feature: Check details
   Scenario: I can change my answer to a question and am returned to the check details page with the correct information
     Given I am on the check details page having entered valid details for a pregnant woman
     When I choose to change my answer to are you pregnant
-    And I select the No option
+    And I select the No option on the are you pregnant page
```

Comment:
This might be better worded as 'And I say No to the are you pregnant question'. The scenario already makes it clear what page we're on

Reply:
Done.